lib/active_model/one_time_password.rb

```diff
@@ -7,13 +7,15 @@ def has_one_time_password(options = {})
         cattr_accessor :otp_column_name, :otp_counter_column_name,
                        :otp_backup_codes_column_name
         class_attribute :otp_digits, :otp_counter_based,
-                        :otp_backup_codes_count, :otp_one_time_backup_codes
+                        :otp_backup_codes_count, :otp_one_time_backup_codes,
+                        :otp_interval
 
         self.otp_column_name = (options[:column_name] || "otp_secret_key").to_s
         self.otp_digits = options[:length] || 6
 
         self.otp_counter_based = (options[:counter_based] || false)
         self.otp_counter_column_name = (options[:counter_column_name] || "otp_counter").to_s
```

Comment:
Not part of this PR but would be good to start to move the default values to class constants

Reply:
I created another PR for this suggestion 🎉 https://github.com/heapsource/active_model_otp/pull/89